readthedocs/projects/forms.py

```diff
@@ -672,8 +672,16 @@ def to_python(self, value):
         """Convert a text area into a list of items (one per line)."""
         if not value:
             return []
-        # Sanitize lines removing trailing spaces and skipping empty lines
-        return [line.strip() for line in value.splitlines() if line.strip()]
+        # Sanitize lines:
```

Comment:
We are not sanitizing paths here, but normalizing only 

```suggestion
        # Normalize paths and filter empty lines:
```